worker/manager/src/mirrors_qa_manager/entrypoint.py

```diff
@@ -0,0 +1,28 @@
+import argparse
+import logging
+import sys
+
+from mirrors_qa_manager import logger
+from mirrors_qa_manager.__about__ import __version__
+from mirrors_qa_manager.worker import WorkerManager
+
+
+def main():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument("id", help="Worker ID")
```

Comment:
Yeah, an env as default is better

Reply:
Thought about it a little more and we should probably leave it explicit. If I let it be an env variable, it would be required and a simple `--help` would require it to be set.